packages/network-of-terms-catalog/catalog/datasets/picot.jsonld

```diff
@@ -0,0 +1,69 @@
+{
+  "@context": "https://schema.org/docs/jsonldcontext.jsonld",
+  "@id": "https://terms.personsincontext.org/ThesaurusHistorischePersoonsgegevens",
+  "@type": "Dataset",
+  "name": [
+    {
+      "@language": "nl",
+      "@value": "Thesaurus Historische Persoonsgegevens"
+    }
+  ],
+  "alternateName": [
+    {
+      "@language": "en",
+      "@value": "PiCoT"
```

Comment:
@RubenSchalk @pwoltjer This is the (common) abbreviation for this source; cf. NTA, GTAA etc. I changed this from `PiCo` to `PicOT` to differentiate the _ontology_ from the _terms list_. I saw @pwoltjer (👋) using `picot:` in the [examples](https://github.com/CBG-Centrum-voor-familiegeschiedenis/PiCo/blob/main/examples/extending-pico/slavenregister.ttl#L7), so `PiCoT` seems to make sense. Agreed?